### ISSUE TITLE
Two Lines Generated

### ISSUE THREAD

**Issue by ecapitaladvisors** [samuelcolvin/watchgod#28]
Samual,

I am learning python free style so I will apologize in advance because this is probably a complete misunderstanding on my part. I used your code below and it works great the only question I have is why this would return two lines. I only copied the superstore xls file into the directory once. Thanks!

![filewatch_run](https://user-images.githubusercontent.com/40032953/53254610-4f474100-3689-11e9-92a2-a4ff6c38714e.png)

```py
import asyncio
from watchgod import awatch

async def main():
    async for changes in awatch('/Users/tgaluska/File'):
        print(changes)

loop = asyncio.get_event_loop()
loop.run_until_complete(main())
```

**Comment by samuelcolvin** [comment]
(in future, it makes code much easier to read if you format it correctly in issues, I've updated your code myself here.)

The short answer is I don't know. Possible reasons:
* the way you copied the file caused multiple file events - this is quite common with things like IDEs, but watchgod is supposed to avoid this problem
* I've never tested watchgod on windows, so it might be a problem particular to windows.

In short, there's not enough information here for me to really answer, you'll need to dig deeper. If you find a bug let me know.
